src/AzureCacheOptions.cs

```diff
@@ -76,13 +73,44 @@ public class AzureCacheOptions
         return ((double)age.Ticks / lifespan.Ticks) > .75; // Refresh if current token has exceeded 75% of its lifespan
```

Comment:
Not related with this PR, but since we are at it...
I wonder why we use 75% here? MSAL uses [5 minutes](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/174da803c69e647820c9da021ef8eabe494efa8a/src/client/Microsoft.Identity.Client/Internal/Constants.cs#L34) as that is the conventional value to contain potential clock skew. Shall we use the same mechanism?

Reply:
We decided to be a little more conservative to withstand interruptions where Entra is down or inaccessible for a while. With a typical 24h token lifetime, refreshing at 75% will let us remain connected even if there's an outage lasting several hours.

Reply:
> With a typical 24h token lifetime,

Good to know it is 24 hours instead of ~1 hour a couple of years ago when I decoded the JWT to check out the "exp" claim. Access token is meant to be short lived; while refresh token can last long to forever.
Good point on the resilient!